### PR TITLE
Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
-Assignment done

### CODE REVIEW

File: src/vaishnaviVJun20/Ex2.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	
+	void Display()
+	{
+		System.out.println("Viahsnavi");
+	}
+
+	public static void main(String[] args) {
+		Ex2 ex2 = new Ex2();
+		ex2.Display();
+
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20/Ex2.java]: Remove space.

File: src/vaishnaviVJun20/Ex2.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package vaishnaviVJun20;
+
+public class Ex2 {
+	
+	void Display()
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20/Ex2.java]: method name should be start with small letter.

Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20/Ex2.java]: Done

File: src/vaishnaviVJun20/Ex2.java (hunk 1 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20/Ex2.java]: Done

**Pull request merged**